docs/3-building-blocks-of-oop-part-2/hometask/src/Bow.ts

```diff
@@ -1 +1,27 @@
-// your code goes here
\ No newline at end of file
+import { Weapon } from "./Weapon";
+
+const MAXIMUM_DURABILITY_MODIFIER_VALUE = 1;
+
+export class Sword extends Weapon {
+  constructor(
+    baseDamage: number,
+    baseDurability: number,
+    value: number,
+    weight: number
+  ) {
+    super("bow", baseDamage, baseDurability, value, weight);
+  }
+
+  polish(): void {
+    if (
```

Comment:
1. Please, use getter and setter for `durabilityModifier` (`durabilityModifier` should be private according to uml)
2. The implementation implies that `durabilityModifier` should be less or equal to 1, but according to the task effective durability should be not greater than 1
3. It is not clear from the task, but let's make `MODIFIER_CHANGE_RATE` static const of the Weapon class

Reply:
done